student/views/registration.py

```diff
@@ -79,6 +79,7 @@ class PaymentPreviewView(View):
     def get(self, request, registration_id):
         registration = Registration.objects.get(pk=registration_id)
         original_payable_amount = settings.CODING_BASICS_REGISTRATION_FEE_SGD
+        early_bird_discounted_amount = registration.batch.current_price()
         discount = 0
```

Comment:
now that we've introduced a new type of discount (early-bird), it might be helpful to change this variable name to `coupon_discount` so that reader can make that distinction.